### PR TITLE
Product of Digits of Sum

### PR DESCRIPTION
**Pull request by VolhaVaitulevich**
Description: https://edabit.com/challenge/Kzmyf4pLx66ZRsnWk

### CODE REVIEW

File: exercises/arrays/digitsOfSum/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+		periodOfArgs = Array.from(String(periodOfArgs), Number).reduce(
+			(acc: number, item: number): number => {
+				acc = acc * item;
+				return acc;
+			},
+			1,
+		);
+	}
+	return periodOfArgs;
+};
```
Comment (siarhei-papkou) [inline exercises/arrays/digitsOfSum/index.ts:9]: Let's try to create a recursive version of this solution.

Comment (VolhaVaitulevich) [reply exercises/arrays/digitsOfSum/index.ts:9]: Done, created a new recursive function (periodOfDigits)

Comment (VolhaVaitulevich) [reply exercises/arrays/digitsOfSum/index.ts:9]: Updated again as we discussed (to have one function)

File: exercises/arrays/digitsOfSum/index.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	if (periodOfArgs < 10) {
+		return periodOfArgs;
+	} else {
+		const periodOfArgsArr = Array.from(String(periodOfArgs), Number);
+		return sumDigProd(
+			periodOfArgsArr.reduce((acc, item) => {
+				acc = acc * item;
+				return acc;
+			}, 1),
+		);
```
Comment (siarhei-papkou) [inline exercises/arrays/digitsOfSum/index.ts]:
```ts
periodOfArgsArr.reduce((acc, item) => acc * item, 1),
```
same for `periodOfArgs`

Comment (VolhaVaitulevich) [reply exercises/arrays/digitsOfSum/index.ts]: Done, updated in a new commit

**Review by siarhei-papkou — APPROVED** [review]

**Pull request merged**